.github/workflows/ci-pr-python.yml

```diff
@@ -1,7 +1,5 @@
-name: TileDB-SOMA Python CI (Minimal)
+name: TileDB-SOMA Python CI (Pull-request)
 
-# This workflow calls ./python-ci-single.yml on a limited subset of the
-# {os} x {python version} matrix.
 on:
   pull_request:
     branches:
```

Comment:
if I read this right, it won't run on PRs stack on top of other branches (e.g., if I have a PR stacked on top of another PR branch).

I think it might be worth running on all PRs, and just using the `paths` filter to control non-essential runs.

Reply:
good catch